Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM postfwd/postfwd:v1.37
+
+LABEL maintainer="Postfwd GeoIp Spam Plugin Maintainer <ondrej.vaskoo@gmail.com>"
+
+ENV PORT=10040
+ENV POSTFWD_ANTISPAM_MAIN_CONFIG_PATH=/etc/postfwd/anti-spam.conf
+ENV POSTFWD_ANTISPAM_SQL_STATEMENTS_CONFIG_PATH=/etc/postfwd/anti-spam-sql-st.conf
+
+# Install dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        libconfig-any-perl \
+        libconfig-tiny-perl \
+        libconfig-ini-perl \
+        libconfig-general-perl \
+        libdbi-perl \
+        libdbd-mysql-perl \
+        libdbd-pg-perl \
+        libgeo-ip-perl \
+        libtime-piece-perl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy binaries into PATH
+RUN cp /opt/postfwd/sbin/postfwd1 /opt/postfwd/sbin/postfwd2 /usr/sbin/
```

Comment:
your are duplicating binaries. here. What about extending PATH?

Reply:
I know and I was thinking about it before. Firstly I wasn't sure what can be in `/opt/postfwd/sbin/` and I didn't want to let user to run files from it. 

But there are only binaries, so you are right, copying is waste of space.

It would be better if original `postfwd` image didn't copy whole git repository and only copy necessary binaries to appropriate PATH directories.

Reply:
I extended PATH in rebase, but I will probably open issue about it in postfwd.